bindings/nodejs/lib/secret_manager/secret-manager-method-handler.ts

```diff
@@ -19,11 +20,17 @@ export class SecretManagerMethodHandler {
      * @param options A secret manager type or a secret manager method handler.
      */
     constructor(options: SecretManagerType | SecretManagerMethodHandler) {
```

Comment:
Should we structure this the same way as the client/wallet just for consistency? this seems kind of janky

Reply:
id like to fix it in a new issue, but youre right!

Reply:
done